src/main/java/com/vonage/quickstart/sms/SendUnicodeMessage.java

```diff
@@ -19,26 +19,26 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package com.nexmo.quickstart.sms;
+package com.vonage.quickstart.sms;
 
-import com.nexmo.client.NexmoClient;
-import com.nexmo.client.sms.SmsSubmissionResponse;
-import com.nexmo.client.sms.SmsSubmissionResponseMessage;
-import com.nexmo.client.sms.messages.TextMessage;
+import com.vonage.client.VonageClient;
+import com.vonage.client.sms.SmsSubmissionResponse;
+import com.vonage.client.sms.SmsSubmissionResponseMessage;
+import com.vonage.client.sms.messages.TextMessage;
 
-import static com.nexmo.quickstart.Util.configureLogging;
-import static com.nexmo.quickstart.Util.envVar;
+import static com.vonage.quickstart.Util.configureLogging;
+import static com.vonage.quickstart.Util.envVar;
 
 public class SendUnicodeMessage {
 
     public static void main(String[] args) throws Exception {
         configureLogging();
 
-        String NEXMO_API_KEY = envVar("NEXMO_API_KEY");
-        String NEXMO_API_SECRET = envVar("NEXMO_API_SECRET");
+        String VONAGE_API_KEY = envVar("VONAGE_API_KEY");
+        String VONAGE_API_SECRET = envVar("VONAGE_API_SECRET");
         String TO_NUMBER = envVar("TO_NUMBER");
 
-        NexmoClient client = NexmoClient.builder().apiKey(NEXMO_API_KEY).apiSecret(NEXMO_API_SECRET).build();
+        VonageClient client = VonageClient.builder().apiKey(VONAGE_API_KEY).apiSecret(VONAGE_API_SECRET).build();
 
         TextMessage message = new TextMessage("Acme Inc", TO_NUMBER, "Blue Öyster Cult \uD83E\uDD18", true);
```

Comment:
Good catch. That was one of those changes that didn't go well with the rebase apparently.
fixed

Reply:
From the code snippet specs, it could be a name or number: 
`The alphanumeric string that represents the name or number of the organization sending the message.`
But I will definitely change it to a constant

Reply:
fixed